### ISSUE TITLE
tests fail on FreeBSD w/ default local

### ISSUE THREAD

**Issue by jmgurney** [xattr/xattr#60]
The default local has a fs encoding of US-ASCII which causes the tests to fail.  A setting of en_US.UTF-8 makes the tests pass (w/ the changes in issue #59 ).  Not sure if setting the local before running tests is best option or not.

**Comment by nyov** [comment]
Is this resolved now after #66 ?

**Comment by jmgurney** [comment]
Yes, I believe it does:
```$python setup.py test
running test
running egg_info
writing requirements to xattr.egg-info/requires.txt
writing xattr.egg-info/PKG-INFO
writing top-level names to xattr.egg-info/top_level.txt
writing dependency_links to xattr.egg-info/dependency_links.txt
writing entry points to xattr.egg-info/entry_points.txt
reading manifest file 'xattr.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'xattr.egg-info/SOURCES.txt'
running build_ext
generating cffi module 'build/temp.freebsd-11.0-RELEASE-p7-amd64-2.7/_lib.c'
already up-to-date
copying build/lib.freebsd-11.0-RELEASE-p7-amd64-2.7/xattr/_lib.so -> xattr
test_attr_fs_encoding_ascii (xattr.tests.test_xattr.TestDir) ... ok
test_attr_fs_encoding_unicode (xattr.tests.test_xattr.TestDir) ... ok
test_setxattr_unicode_error (xattr.tests.test_xattr.TestDir) ... ok
test_symlink_attrs (xattr.tests.test_xattr.TestDir) ... ok
test_attr_fs_encoding_ascii (xattr.tests.test_xattr.TestFile) ... ok
test_attr_fs_encoding_unicode (xattr.tests.test_xattr.TestFile) ... ok
test_setxattr_unicode_error (xattr.tests.test_xattr.TestFile) ... ok
test_symlink_attrs (xattr.tests.test_xattr.TestFile) ... ok
test_attr_fs_encoding_ascii (xattr.tests.test_xattr.TestFileWithSurrogates) ... skipped 'Files with invalid encoded names are only supported under linux'
test_attr_fs_encoding_unicode (xattr.tests.test_xattr.TestFileWithSurrogates) ... skipped 'Files with invalid encoded names are only supported under linux'
test_setxattr_unicode_error (xattr.tests.test_xattr.TestFileWithSurrogates) ... skipped 'Files with invalid encoded names are only supported under linux'
test_symlink_attrs (xattr.tests.test_xattr.TestFileWithSurrogates) ... skipped 'Files with invalid encoded names are only supported under linux'

----------------------------------------------------------------------
Ran 12 tests in 0.008s

OK (skipped=4)
```

Though I should spend some time make sure that the skip reason is correct.
